src/main/java/com/spotify/github/jackson/UserSerializer.java

```diff
@@ -0,0 +1,48 @@
+/*-
```

Comment:
This serializer seems like it could be replaced by Jackson annotations too - thoughts?